N3.js/src/main/scala/org/w3/banana/n3js/N3.scala

```diff
@@ -0,0 +1,194 @@
+package org.w3.banana
+package n3js
+
+import scala.scalajs.js
+
+object N3 extends js.Object {
+
```

Comment:
It's called N3.js because this was my initial (and ultimate) implementation goal; however, as the [README](https://github.com/RubenVerborgh/N3.js/tree/v0.4.1#specifications) says:
 
> Note that the library does not support full Notation3 yet (and a standardized specification for this syntax is currently lacking).

I expect full N3 format somewhere this summer; although I should be careful with promises :smile:
The current parsed language is a superset of N-Quads and TriG—N3 will be supported some day.